README.md

```diff
@@ -151,6 +153,10 @@ You could use your browser to confirm this. If everything looks OK, you can shar
 
 ### Publish Your Change to Your Github Account
 
+If you have walked through the optional steps to remove CNAME or tweak the url of `_config.yml`, please revert the change.
+This is because these files, `CNAME` and `_config.yml`, of the target branch, gh-pages of cesehub repository, should not be changed to make the `cesehub.org` keep on working normally.
+We only focus on modifying the contents of this website, use `dev` branch to host the change of the website contents, and use `gh-pages` of your repository to check your content modification.
+
```

Comment:
this session is the only difference from PR https://github.com/cesehub/cesehub/pull/23